dynamic_programming/coin_change.ts

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
This comment should be on the function, not on the interface; not sure if JSDoc can assign it properly otherwise.

Reply:
@appgurueu done